src/main/flight/pid.c

```diff
@@ -344,6 +344,12 @@ static void pidApplyRateController(const pidProfile_t *pidProfile, pidState_t *p
         pidState->errorGyroIfLimit = ABS(pidState->errorGyroIf);
     }
 
+#ifdef USE_SERVOS
+    if (STATE(FIXED_WING)) {
```

Comment:
We probably also need to check that `pidProfile->fixedWingItermThrowLimit` is not zero (zero = disabled as with low-pass filtering)